### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by abretaud**
So, here are a few changes I made while playing with the latest apollo version

    - Added user activate/inactivate
    - Added get_creator for user, group and organisms
    - Added omitEmptyOrganisms to get_users
    - Added support for group admins
    - Repaired GFF3/Fasta downloading

Most of it is not yet very well tested, except the last one. But they are simple functions, so there is not a lot of risk I guess.

### CODE REVIEW

**Comment by abretaud** [comment]
By the way, is it intended that https://python-apollo.readthedocs.io is not using the default theme?

**Comment by hexylena** [comment]
> By the way, is it intended that python-apollo.readthedocs.io is not using the default theme?

eh not really, I remember vaguely changing it but happy for it to be more default

**Comment by abretaud** [comment]
Ok I think this one is ready, do have some time to look at it @erasche (not top priority for me)?
So I've changed the doc to the default theme, but it's hard to test what it will look like at readthedocs before merging. Maybe there will need to make some little change in the readthedoc conf, I can have a look

File: setup.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
+    [x.replace('/', '.') for x in glob.glob('apollo/*') if not x.endswith('.py') and not x.endswith('.pyc')] + ['arrow.commands']
 
 setup(
     name="apollo",
-    version='3.0.4',
-    description="WebApollo API library",
+    version='3.1',
+    description="Apollo API library",
     long_description=readme,
     author="E. Rasche",
```
Comment (hexylena) [inline setup.py]:
```suggestion
    author="Helena Rasche;Anthony Bretaudeau",
```

File: docs/conf.py [hunk 2]
```diff
@@ -1,159 +1,31 @@
...
-#
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
-
-# The master toctree document.
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.napoleon']
 master_doc = 'index'
 
-# General information about the project.
-project = 'Python-Apollo'
```
Comment (hexylena) [inline docs/conf.py:51]: is it fine without project title?

**Review by hexylena — DISMISSED** [review]

File: docs/conf.py (hunk 2 above)
Comment (abretaud) [reply docs/conf.py:51]: I think it's not needed, readthedocs must be guessing it automatically.
I mostly copied the conf.py from python-chado and python-tripal

Comment (hexylena) [reply docs/conf.py:51]: sounds good

**Review by hexylena — APPROVED** [review]

**Pull request merged**